openstack/train/ovs-kernel/nova_os_vif_util.patch

```diff
@@ -24,4 +24,118 @@ index 9b76ef3..c1e3806 100644
      elif _is_firewall_required(vif) or vif.is_hybrid_plug_enabled():
          obj = _get_vif_instance(
              vif,
-
+diff --git a/nova/virt/libvirt/config.py b/nova/virt/libvirt/config.py
```

Comment:
you push it to train release? I thoght we did it on Ussuri release. ..

Reply:
@moshe010  I did for both, there is two files one for ussuri and the other for train